pynetdicom/ae.py

```diff
@@ -37,6 +37,9 @@ class ApplicationEntity(object):
         messages. A value of ``None`` means no timeout. (default: ``30``)
     ae_title : bytes
         The local AE's *AE title*.
+    connection_timeout : int or float or None
+        The maximum amount of time (in seconds) to wait for a TCP connection to be
+        established. A value of ``None`` means no timeout. (default: ``None``)
```

Comment:
Might be good to link to the Python `socket.settimeout` docs here.

Reply:
done